tao_compiler/mlir/xla/ral/context/stream_executor_based_impl.cc

```diff
@@ -1392,6 +1392,9 @@ Status RunCudnnConvolution<int8_t>(
     se::DeviceMemoryBase& result_buffer,
     se::ScratchAllocator* scratch_allocator, se::Stream* stream,
     se::dnn::ProfileResult* profile_result) {
+#if defined(TF_IS_DEEPREC)
```

Comment:
DeepRec doesn't support `RunCudnnConvolution` ?

Reply:
DeepREC is tf-1.15, which does not support `RunCudnnConvolution` with `int8_t` type.